### DISCUSSION TITLE
Help for Thegraph

### DISCUSSION THREAD

**Discussion by Leopard-CC** [community/community#33795] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm facing problem with deploying subgraph
I just followed the guide [https://thegraph.com/docs/en/deploying/deploying-a-subgraph-to-studio/](guide)
But not worked on my side
Hope your help

**Comment by cryptofan430** [comment, marked as answer]
Once you built the graph on your local you need to create new subgraph  on [https://thegraph.com/explorer/dashboard](thegraph)
After then you should authenticate locally 
`npx graph auth https://api.thegraph.com/deploy/ <ACCESS_TOKEN>`

Finally you can deploy
`npm run deploy`

Best luck

**Reply by Leopard-CC** [reply]
Thank you guy
That's what I'm looking for
